internal/pkg/manifest/workload.go

```diff
@@ -25,25 +26,47 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// AWS VPC subnet placement options.
 const (
-	defaultFluentbitImage = "amazon/aws-for-fluent-bit:latest"
-	defaultDockerfileName = "Dockerfile"
-
-	// AWS VPC subnet placement options.
 	PublicSubnetPlacement  = "public"
 	PrivateSubnetPlacement = "private"
 )
 
+// Platform options.
+const (
+	OSLinux                 = dockerengine.OSLinux
+	OSWindows               = dockerengine.OSWindows
+	OSWindowsServer2019Core = "windows_server_2019_core"
```

Comment:
Q: I understand why the OSFamily consts live in the manifest package as opposed to the dockerengine package, but why do we redefine OSLinux and OSWindows again here? Is it for visibility/to be with the other OS consts?

Reply:
I debated this too! And then I settled that each package should be responsible for its own set of inputs.
- `dockerengine` is a library and its constants should match 1:1 the platform specifications that Docker allows.
- `template` surfaces only platform options that AWS services can support.
- `manifest` can control which values of `dockerengine` we want to allow so that it works with `template`.

The main benefit of redefining it I think is for the clients. This way `stack.convertPlatform` does have to import both `dockerengine` and `manifest` to figure out what `manifest` allows. Instead, it will have every allowed value it needs in `manifest`.

The std lib does something similar with `filepath.Separator` for example: https://pkg.go.dev/path/filepath#pkg-constants 

Hope that makes sense!